Game/Templates/Source

```diff
@@ -1,5 +1,9 @@
 #include "{0}.h"
 
+EXPOSE_MEMBERS({0}) {1}
+
+{2};
+
 GENERATE_BODY_IMPL({0});
 
 void {0}::Start() {1}
```

Comment:
For my own awareness in future reviews, here is how you updated the templates of a script file right?

(I mean, when creating a script, now this macro will be automatically generated in the .cpp)